src/pool/srv_layout.h

```diff
@@ -26,7 +26,7 @@
 #include <daos_types.h>
 
 /* Default layout version */
-#define DS_POOL_MD_VERSION 4
+#define DS_POOL_MD_VERSION 5
```

Comment:
Will this conflict with #7632?

Reply:
Yup, i intent to rebase one of PR once other one merged, Or do you think it makes sense i bump this one to 6 directly in this PR.